data/mods/Magiclysm/hobbies.json

```
@@ -87,7 +87,7 @@
     "name": "Dabbler (Peaceful Rest)",
     "description": "You were one of the hundreds of millions of people who knew a single spell.  In your case, Peaceful Rest.  Ever since you were a child, you dreaded bedtime.  Falling asleep was an ordeal, a battle between you and the bed, and it seemed like no matter how many sheep you counted, sleep hygiene habits you practiced, or exhausted you felt, it was an hour or more until you final fell into sleep.  You became a mage because you were tired of having 5% of your life stuck staring at the ceiling while waiting for sleep to come, and ever since the world ended, you're even more grateful you can fall asleep whenever you need to.",
```

```suggestion
    "description": "You were one of the hundreds of millions of people who knew a single spell.  In your case, Peaceful Rest.  Ever since you were a child, you dreaded bedtime.  Falling asleep was an ordeal, a battle between you and the bed, and it seemed like no matter how many sheep you counted, sleep hygiene habits you practiced, or exhausted you felt, it was an hour or more until you finally fell into sleep.  You became a mage because you were tired of having 5% of your life stuck staring at the ceiling while waiting for sleep to come, and ever since the world ended, you're even more grateful you can fall asleep whenever you need to.",
```

~~Shouldn't this grant insomnia as a trait if the description says you have it~~